### PR TITLE
Disruption: fix some bugs on the whole at chain

### PR DESCRIPTION
**Pull request by antoine-de**

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/fields.py [hunk 1]
```diff
@@ -380,6 +379,7 @@ def output(self, key, obj):
     "messages": NonNullList(NonNullNested(disruption_message)),
 }
 
+
```
Comment (l-vincent-l) [inline source/jormungandr/jormungandr/interfaces/v1/fields.py:382]: 2 new lines???

File: source/type/task.proto [hunk 2]
```diff
@@ -11,5 +11,8 @@ enum Action{
 
 message Task{
     required Action action = 1;
+
+    optional pbnavitia.realtime.Message message = 3;
```
Comment (l-vincent-l) [inline source/type/task.proto:15]: Why don't you use 2?

File: source/jormungandr/jormungandr/interfaces/v1/fields.py (hunk 1 above)
Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/fields.py:382]: yes it is pep8 compliant

File: source/jormungandr/jormungandr/interfaces/v1/fields.py [hunk 3]
```diff
@@ -367,7 +367,6 @@ def output(self, key, obj):
     "text": fields.String(),
     "content_type": fields.String(),
 }
-
```
Comment (l-vincent-l) [inline source/jormungandr/jormungandr/interfaces/v1/fields.py:370]: it's not pep8 compliant

**Pull request merged**